experiment/textcov.py

```diff
@@ -396,9 +396,8 @@ class name specification use single upper case letter for
         if c == 'L':
           start = True
           if next_arg:
-            if array_count > 0:
-              next_arg = f'{next_arg}{"[]" * array_count}'
-              array_count = 0
+            next_arg = f'{next_arg}{"[]" * array_count}'
```

Comment:
Could we change this to `next_arg += f'{"[]" * array_count}'` -- isn't it a bit easier to read?